data/yam/autoyast/support_images/autoyast_sles15sp6_media_install_textmode_default_patterns_s390x.xml.ep

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
```

Comment:
Not a dynamic profile any more, remove the extension '.ep'.

Reply:
bc7990acb